client/state/invites/test/selectors.js

```diff
@@ -0,0 +1,48 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { isRequestingResend } from '../selectors';
+
+describe( 'selectors', () => {
+	describe( '#isRequestingResend()', () => {
+		test( 'should return true when requesting resend', () => {
+			const state = {
+				invites: {
+					requestingResend: {
+						12345: { '123456asdf789': false },
+						67890: { '789lkjh123456': true },
+					},
+				},
+			};
+			expect( isRequestingResend( state, 67890, '789lkjh123456' ) ).to.equal( true );
+		} );
+		test( 'should return false when resend request is complete', () => {
+			const state = {
+				invites: {
+					requestingResend: {
+						12345: { '123456asdf789': false },
+						67890: { '789lkjh123456': true },
+					},
+				},
+			};
+			expect( isRequestingResend( state, 12345, '123456asdf789' ) ).to.equal( false );
+		} );
+		test( 'should return false when resend has not been requested', () => {
+			const state = {
+				invites: {
+					requestingResend: {
+						12345: { '123456asdf789': false },
+						67890: { '789lkjh123456': true },
+					},
```

Comment:
This should be `requestingResend: {}`, no?

Reply:
When I left this comment, I missed that the invite ID tested here is different, but I made this change in https://github.com/Automattic/wp-calypso/pull/21845 to improve the clarity of this test.